packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/privateStyle..st

```diff
@@ -2,5 +2,6 @@ parsing
 privateStyle: aText
 	
 	| attributes |
+	aText runs: (RunArray runs: {aText size} values: {{}}).
```

Comment:
This line is the one which removes the previous attributes. I couldn't find a cleaner way to do this. If you have any suggestions please do tell.

Sadly we don't have a Text where we can get `runs` from to simply overwrite the `runs` of `aText` like we can in `MarkdownBlockTextStyler>>privateStyle`

Reply:
How about:
`aText addAttribute: TextEmphasis normal.`?

Reply:
Thanks